packages/components/src/Radio/Radio/_docs/Radio.stickersheet.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Radio } from "../index"
+
+export default {
+  title: "KAIO-staging/Radio controls/Radio",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Built with no label on purpose, to be used within `RadioField` where label is present
+            id: "label",
+            enabled: false,
+          },
+        ],
+      },
+    },
+  },
```

Comment:
This might be a good way to document something that's a feature not a bug? If we add `aria-label` it makes it feel like something that is a legit pattern to follow and not something we do just for stickersheets

Reply:
Good point! Yeah I'm on board with this approach :) Worth documenting.